.github/workflows/windows-dll.yml

```
@@ -0,0 +1,30 @@
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build
+    runs-on: windows-latest
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v4
+    - name: Build windows artifacts
+      shell: powershell
+       run: |
```

actionlint (1.7.3)

could not parse as YAML: yaml: line 28: mapping values are not allowed in this context

(syntax-check)